CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ Data Layer Observer follows semantic versioning when releasing updates.
 
 ## History
 
+### 2.1.1
+
+- Prevent the telemetry provider from sending a Rule Registration metric if there are no rules registered.
```

Comment:
@ndonze-fs, i added both PR's to the changelog. lmk if the other PR should increment the version number to 2.1.2 and these should be broken up.